src/explore-education-statistics-common/src/components/form/rhf/__tests__/RHFFormTextArea.test.tsx

```diff
@@ -0,0 +1,243 @@
+import RHFFormTextArea from '@common/components/form/rhf/RHFFormTextArea';
+import FormProvider from '@common/components/form/rhf/FormProvider';
+import { render } from '@testing-library/react';
+import noop from 'lodash/noop';
+import React from 'react';
+
+describe('RHFFormTextArea', () => {
+  test('renders correctly with required props', () => {
+    const { container, getByLabelText } = render(
```

Comment:
We could use the newer `screen` selector instead i.e. `screen.getByLabelText`

Think we can refactor the rest of this suite to use this instead as well.